### PR TITLE
Add web tab to admin page

### LINKED ISSUE #40: Add web tab
# TODO
- [x] Committee
- [x] Sponsors
- [x] Tournaments
- [x] FAQ

### PR DESCRIPTION
**Pull request by Fastjur**
- [x] Requires https://github.com/AreaFiftyLAN/lancie-frontend/pull/517 to be merged too.

Closes #40 

Blocked by:
- [x] https://github.com/AreaFiftyLAN/lancie-api/pull/457
- [x] https://github.com/AreaFiftyLAN/lancie-api/pull/458

# This PR adds the following

### A webtab for changing the frontend sponsors, tournaments, FAQ items and committee
image

### Adding a committee member dialog
image

### Adding a FAQ item dialog
image

### Adding / editing a tournament dialog
image  
I have some issues with this dialog. Mainly due to the way we specify the image right now. This should be changed to an image clipper and uploader in the future. And also the fact that the description usually is very long and is markdown. We could make this a live editor perhaps? But that would require more space, probably a separate page

### Adding / editing a sponsor dialog
image

### Deletion confirmation dialog
image

### CODE REVIEW

File: src/lancie-admin-web/lancie-admin-committee.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      'use strict';
+      Polymer({
+        is: 'lancie-admin-committee',
+        properties: {
+          committee: Object
+        },
+
+        deleteMember: function(e) {
+            this.committee = this.committee.filter(function(member) {
+              console.log(e, member);
```
Comment (martijnjanssen) [inline src/lancie-admin-web/lancie-admin-committee.html]: Don't forget to remove this one and the one underneath.

**Comment by martijnjanssen** [comment]
Would it be good to use a `lancie-dialog` and put the field for adding a user in there? Don't want to clutter the UI with things you don't always need.

File: src/lancie-admin-layout/lancie-admin-layout.html [hunk 2]
```diff
@@ -78,6 +78,10 @@
         border-radius: 5px;
         padding: 5px;
       }
+
+      .web {
+        border-top: 1px solid #e8e8e8;
```
Comment (martijnjanssen) [inline src/lancie-admin-layout/lancie-admin-layout.html]: It's literally in line 72...

Comment (Fastjur) [reply src/lancie-admin-layout/lancie-admin-layout.html]: Aah woops this was due to refactoring. Missed that!

**Comment by elarb** [comment]
Looks like this PR isn't blocked anymore?

**Comment by Fastjur** [comment]
Correct @elarb 

I'll fix the conflicts and then it should be ready :+1:

**Review by skamoen — DISMISSED** [review]
Tested:
- Remove tournaments :+1: 
- Remove FAQ :+1: 
- Add FAQ :+1: 
- Add Tournament
- Edit Sponsor
- Add Sponsor

- Edit Tournament :-1: : No endpoint on api?
`{"timestamp":"2018-10-07T16:58:53Z","status":405,"error":"Method Not Allowed","message":"Request method 'POST' not supported","path":"/api/v1/web/tournament/211"}`

**Review by Fastjur — COMMENTED** [review]
Tested:

Remove tournaments 👍

Remove FAQ 👍

Add FAQ 👍

Add Tournament 👍 

Edit Sponsor 👍 

Add Sponsor 👍 

Edit Tournament 👍

**Review by timhuisman-dev — APPROVED** [review]

**Pull request merged**